src/resources/develop/Currencies.yaml

```diff
@@ -1749,13 +1756,20 @@ values:
         - VCT
       monetary_authority: Eastern Caribbean Central Bank
 
+  - code: XDR
+    name: IMFSpecialDrawingRights
+    label: IMF special drawing rights
+      metadata:
+        symbol: "SDR"
+        precision: 2
+        monetary_authority: International Monetary Fund
```

Comment:
Do we want people to be able to create an “IMF special drawing rights credit” instrument? Would that even have a useful meaning? Any currency that’s defined here (and has an exchange rate, which this does I think) will currently be listed in the CDN currency selector

Reply:
Resolution: should be added to this list for completeness. Can add separate filtering for CRN selection.